docs/connect.html

```diff
@@ -32,7 +32,7 @@ <h3><a id="connect_overview" href="#connect_overview">8.1 Overview</a></h3>
 
     <h3><a id="connect_user" href="#connect_user">8.2 User Guide</a></h3>
 
-    <p>The quickstart provides a brief example of how to run a standalone version of Kafka Connect. This section describes how to configure, run, and manage Kafka Connect in more detail.</p>
+    <p>The <a href="../quickstart">quickstart</a> provides a brief example of how to run a standalone version of Kafka Connect. This section describes how to configure, run, and manage Kafka Connect in more detail.</p>
```

Comment:
The first time reader may not find where the **quickstart** is for the example.
before:
![image](https://user-images.githubusercontent.com/43372967/82001847-bbe39c80-968e-11ea-9de9-0bb5bd870cd6.png)
after:
![image](https://user-images.githubusercontent.com/43372967/82001879-d3bb2080-968e-11ea-9bd3-6d072fe9b3e8.png)